docs/installation.rst

```diff
@@ -48,6 +48,12 @@ Use ``pip`` to install julearn from `PyPI <https://pypi.org>`_, like so:
 
     pip install junifer
 
+You can also install via ``conda``, like so:
```

Comment:
Same as above: Would it make sense to let users know that there may be some issues with git-annex/datalad on macs?